funding/manager_test.go

```diff
@@ -1041,11 +1041,12 @@ func assertAddedToRouterGraph(t *testing.T, alice, bob *testNode,
 // confirmed. The last arguments can be set if we expect the nodes to advertise
 // custom min_htlc values as part of their ChannelUpdate. We expect Alice to
 // advertise the value required by Bob and vice versa. If they are not set the
-// advertised value will be checked against the other node's default min_htlc
-// value.
+// advertised value will be checked against the other node's default min_htlc,
+// base fee and fee rate values.
 func assertChannelAnnouncements(t *testing.T, alice, bob *testNode,
 	capacity btcutil.Amount, customMinHtlc []lnwire.MilliSatoshi,
-	customMaxHtlc []lnwire.MilliSatoshi) {
+	customMaxHtlc []lnwire.MilliSatoshi, baseFees []lnwire.MilliSatoshi,
```

Comment:
nit: maybe we could add here something like

```
type channelAnnouncementCheck func(msg *lnwire.Message) error

func assertChannelAnnouncements(t *testing.T, alice, bob *testNode,
capacity btcutil.Amount, customMinHtlc []lnwire.MilliSatoshi,
customMaxHtlc []lnwire.MilliSatoshi, additionalChecks ...ChannelAnnouncementCheck) {
    // ...
    for _, check := range additionalChecks {
        check(msg)
    }
}
```

So we don't have to pass most of the checks when they are not needed. It can be added later so no worries.

Reply:
Yes it makes sense to condense the parameter list of `assertChannelAnnouncements` the way you describe. We could   probably take `customMinHtlc`, `customMaxHtlc`, `baseFees` and `feeRates` and put them into a `htlc.ForwardingPolicy` for each Alice and Bob and pass it to the `additionalChecks` function where necessary. 

If you agree then I would keep the test as is in this PR and tackle a refactoring of it in a follow-up PR.